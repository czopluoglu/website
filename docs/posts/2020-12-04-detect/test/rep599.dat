11111100000110101100
00001000111111101101
10001000011001010101
11100111101011010101
01111101111111101111
00000000101010100100
00001100101000000100
11111111101001101111
01001111011111000000
00101101110000111110
00000000011110100010
11101101111011101111
10010011110011100111
10010100011000110011
10111101110010101011
00000111110111110110
00011001001010100010
11110111110111101111
11100101110111111110
00001001000100100010
10100011001010000110
11111100101110010111
11100001011111100011
01001111000010010011
11100101111101100111
11001001101110110111
01001000001110110101
01010101001110111010
11101110111000101011
10010011011111111111
11100101101010101100
11111011101101111111
11111101111100001011
11111110111010110110
00100001100100001100
10000100101010100110
10010111110010010110
01110101011101010110
10001001100010011010
10001001101001000010
01111110111111111011
10011111111011011100
01010000101110001000
11001001111000101111
01110001101000110110
01100111111001111111
10010110111000011000
00000011000001100000
10010100111010111110
11110100110010111101
00110111011011001111
10111001011111111111
01000101111010100100
11001001101010001010
01011100010111100010
00000000011100011110
00000001101000100011
01110111101101111111
10100101001011000011
10001000000100001011
00001110111010111111
00110001011111101111
01110001110111000110
00100001011010100111
10010100101110111111
00001001111001101011
10111101111011111110
11110011110010100010
10010111111011100101
00100001001111110111
00001001000000101000
01000101011110101111
11010100101110110010
11001101111011011111
11000001011011110110
00001010011111111101
11001101101011011011
00000001001011010110
00010111010110110111
00101000011101101100
00111111111101011111
00000000001110001010
00111100001111100101
10001001010000101111
11100111001111111111
00000001000001100010
11000001000010110110
10011111001010100011
11101100101010100001
10111001001011111111
11110011111111011100
11010111010111001111
11011011100011110111
01001110011101111011
11111111101101001111
00110110011111101011
00000101000001100110
10101000001110101010
00011001001110101111
11011001100111010110
00101111011101101100
10111101111111011110
10111110111111001111
01011111111010111111
01110011110001010101
11000101101110010010
01100001101100110110
11111101101100111100
11010110111000110010
11010101111111101111
11010010111100100111
00000101101001111111
10111101111101111011
11100101111010110110
00001100100100000010
10011000001110101111
00000100101000101111
11010101111111011110
11110111011010101111
01010000011110100000
11011100111110110011
11101101110011111111
11110111111011001011
01010111001100000111
01000001001010011111
00010001101111101110
00000000011001111000
00001001101010101110
10101101011110111101
00001000101011011111
10100001100101110111
01010100000011100111
00000100111011100010
11110111111001111111
00110011011111100001
11011111000100001100
01011101011001000000
11100001011110110111
10001001000100000110
11000111111100100100
10110000011110000111
11000110001001000101
01100101011010110011
10101110101010110111
00001101111000010011
10101011111110101101
10101101000001101111
01000100011100110110
01100001010101110110
10001110101010101110
10001101011001100110
11010000111110011011
10100000001101110111
10101001111011110111
11101011111111111111
00000000000011010010
10001110111100111111
01111001100010100000
10001001010110000110
00000100000010111101
01101001101000000101
10111110111001000110
11110111101010010111
00001101000100101111
10101111101110101111
11110011101001000011
10111011111110101110
11001001011110101111
11000101101011110111
01111010111110101111
01011000001001010110
00000011101100000010
10111111111011101011
10001101110101101110
10111111111111110110
11101110111011011111
00110000111001001010
00001010011101100111
10000100110101111110
11110110101101101110
10111110111110111010
00000100001001100001
00001001001110011110
00011001011010000001
01010001001000111111
00010001001110011110
11011010010010001100
10100101111100001100
10100001101101010010
01011011011111011100
10001001011010100000
11001101000101110010
01000001111110100110
11001100111101000001
01011000100110100110
00000010001111111111
00000000010011100111
11011011111110110111
10011001110000000011
11011110111011101110
01101000101111000010
10011110111000101111
10000001101110000011
00110111110010100111
10110101101011111110
00101001011110011111
00010011011110100111
10001001101111111111
11000101010010011110
01100101011110000011
10101111110000101010
11100101001110001111
10111001111010110011
01011000111101100110
11010000110100000010
10111001101011111101
01010101110010000010
10000101101111010011
11101100101111100110
00110001011011010010
10110011111111001111
00000101101111111011
10010011011010000101
01111100010110001000
10001011111000100010
11010111111100001101
01001101111110000111
00100001111001101111
11101010011101111111
11101110101111111111
11101101011110011110
11111110011111101111
10000010111111111101
01000100010000100010
10101001101110011011
11010000011110111111
01010101111010001110
01101001001111000011
00011000000100000110
00010111010000100010
01110101001110011111
00000001011111000001
10001000111101101110
10001100001111101101
10110111111111110110
00101000111111001111
11000110111111011111
01011101111110111111
10110100011010100001
10011111110000011110
