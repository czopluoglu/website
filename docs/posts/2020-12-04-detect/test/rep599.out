-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep599.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.20

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.65
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7458
                   IDN index value:   0.7158
                           Ratio r:   0.6281

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    2    1    3    1    1    4    1    1
    2    2    2    2    3    4    3    3    4    2

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    4    6    7    9   10

   -----------CLUSTER  2-------------

    3   11   12   13   14   20

   -----------CLUSTER  3-------------

    5   15   17   18

   -----------CLUSTER  4-------------

    8   16   19
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d--+++-+-----+-----+
   -d++-+---++------++-
   -+d+++++--+---------
   +++d+++-------++----
   +-++d++----++---+---
   +++++d-++----+---+++
   --+++-d----+--++----
   +-+--+-d++-++----+--
   -----+-+d++++--+--+-
   -+-----++d++-+-++--+
   -++-----++d-+----++-
   ----+-++++-d+---+-+-
   ----+--++-++d++++--+
   +----+---+--+d++----
   ---+--+-----++d-+--+
   ---+--+-++--++-d+-+-
   ----+----+-++-++d++-
   -+---+-+--+-----+d++
   -+---+--+-++---+++d+
   +----+---+--+-+--++d
 
 No cross validation for this DETECT run
